USING_ELEKTRA.md

```diff
@@ -0,0 +1,136 @@
+# Using Elektra
+
+In the new version of lcdproc, [Elektra](https://www.libelektra.org/home) 
+will be used to store and edit configuration settings which are used by lcdproc.
+
+Elektra stores configuration in a global, hierarchical key database. Elektra
+comes with a lot of benefits such as [code generation](https://www.libelektra.org/tools/gen) or
+[advanced GUIs](https://www.libelektra.org/tools/qt-gui). But one major benefit
+ is the possibility to have a 
+ [validation on configuration settings](https://www.libelektra.org/tutorials/validate-configuration)
+ because of configuration specifications.
+
+This tutorial will teach you how to introspect such a specification and how
```

Comment:
to change configuration and introspect the specification.

What is the specification?